app/templates/organizations/organization/local-involvements/edit.hbs

```diff
@@ -91,13 +94,8 @@
           <td>
             {{#let involvement.error.involvementType.message as |errorMessage|}}
               <div class={{if errorMessage "ember-power-select--error"}}>
-                {{#if this.isWorshipService}}
-                  {{#if
-                    (eq
-                      this.municipalityCode
-                      involvement.administrativeUnit.classification.id
-                    )
-                  }}
+                {{#if @model.organization.isWorshipService}}
+                  {{#if involvement.administrativeUnit.isMunicipality}}
                     <PowerSelect
                       @noMatchesMessage="Geen resultaten"
                       @options={{this.model.involvementTypes}}
```

Comment:
**Nitpick:** Instead of defining the component twice, the if statement of `involvement.administrativeUnit.isMunicipality` can be inline in the `@options` value